src/components/AddBook.jsx

```diff
@@ -2,44 +2,56 @@
 import React, { useState } from "react";
 import { useDispatch } from "react-redux";
 import { v4 as uuidv4 } from "uuid";
-import { addedBook } from "../redux/books/books";
+import { addBook } from "../redux/books/books";
 
 const AddBook = () => {
   const dispatch = useDispatch();
-  const [initState, setState] = useState({ title: "", author: "" });
+  const [inputState, setState] = useState({
+    title: "",
+    author: "",
+    category: "",
+  });
   const changeState = (e) => {
-    e.preventDefault();
-    setState({ ...initState, [e.target.name]: e.target.value });
+    setState({ ...inputState, [e.target.name]: e.target.value });
   };
 
   const bookState = (e) => {
     e.preventDefault();
-    if (!initState.title.trim() || !initState.author.trim()) return;
+    // if (!inputState.title.trim() || !inputState.author.trim()) return;
```

Comment:
- [ ] [OPTIONAL] Please remove commented code as it makes your codebase look untidy 👍